src/vos/evtree.c

```diff
@@ -2579,6 +2579,14 @@ evt_ent_array_fill(struct evt_context *tcx, enum evt_find_opc find_opc,
 				 * change while we decide how to handle this
 				 * properly.
 				 */
+				if (range_overlap == RT_OVERLAP_INCLUDED &&
```

Comment:
The comment above should be below this

Reply:
ok, the change included by https://github.com/daos-stack/daos/pull/9903, thx